_posts/2022-08-17-release-1.15.2.md

```diff
@@ -0,0 +1,166 @@
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28878'>FLINK-28878</a>] -         PipelinedRegionSchedulingITCase.testRecoverFromPartitionException failed with AssertionError
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28880'>FLINK-28880</a>] -         Fix CEP doc with wrong result of strict contiguity of looping patterns
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28908'>FLINK-28908</a>] -         Coder for LIST type is incorrectly chosen is PyFlink
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28978'>FLINK-28978</a>] -         Kinesis connector doesn&#39;t work for new AWS regions
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28994'>FLINK-28994</a>] -         Enable withCredentials for Flink UI
+</li>
+</ul>
+                
+<h2>        Improvement
+</h2>
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-27199'>FLINK-27199</a>] -         Bump Pulsar to 2.10.0 for fixing the unstable Pulsar test environment.
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-27865'>FLINK-27865</a>] -         Add guide and example for configuring SASL and SSL in Kafka SQL connector document
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28094'>FLINK-28094</a>] -         Upgrade AWS SDK to support ap-southeast-3 
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28140'>FLINK-28140</a>] -         Improve the documentation by adding Python examples
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28486'>FLINK-28486</a>] -         [docs-zh] Flink FileSystem SQL Connector Doc is not right
+</li>
+</ul>
+                           
+<h2>        Sub-task
+</h2>
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-23944'>FLINK-23944</a>] -         PulsarSourceITCase.testTaskManagerFailure is instable
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-24872'>FLINK-24872</a>] -         PulsarSourceITCase.testMultipleSplits failed on AZP
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-25815'>FLINK-25815</a>] -         PulsarSourceITCase.testTaskManagerFailure failed on azure due to timeout
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-25884'>FLINK-25884</a>] -         PulsarSourceITCase.testTaskManagerFailure failed on azure
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-26177'>FLINK-26177</a>] -         PulsarSourceITCase.testScaleDown fails with timeout
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-26721'>FLINK-26721</a>] -         PulsarSourceITCase.testSavepoint failed on azure pipeline
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-27833'>FLINK-27833</a>] -         PulsarSourceITCase.testTaskManagerFailure failed with AssertionError
```

Comment:
These issues are unstable tests about Pulsar, maybe not listed here?

Reply:
Thanks, I have removed the IT test failure fix issues from the release notes